server/src/main/java/org/elasticsearch/search/aggregations/bucket/geogrid/GeoGridAggregationBuilder.java

```diff
@@ -65,7 +64,7 @@
 
     public static ObjectParser<GeoGridAggregationBuilder, Void> createParser(String name, PrecisionParser precisionParser) {
         ObjectParser<GeoGridAggregationBuilder, Void> parser = new ObjectParser<>(name);
-        ValuesSourceParserHelper.declareGeoFields(parser, false, false);
+        ValuesSourceAggregationBuilder.declareFields(parser, false, false, false);
```

Comment:
Probably a thing for later more than now: three boolean parameters can be kind of hard to keep track of. I think IntelliJ has something where it puts the names of parameters but that isn't universal. Anyway, *something* else might be a little more descriptive. I'm not sure what though. It certainly can wait for later too.

Reply:
I went back and forth on that.  The other obvious alternative is to have a `declareTimeZoneAwareField` and `declareTimeZoneUnawareField` methods, which would replace the third boolean.  But I kind of hate moving parameters into method names, so I went with this one.  I agree it's hard to keep track of though.  I suppose we could make it a small builder, but that seems like too much.  IDK.  Open to suggestions.